specs/v2-sketch.yaml

```diff
@@ -524,3 +554,52 @@ components:
           items:
             type: string
             format: uuid
+    TriggerSummary:
+      type: object
+      properties:
+        type:
+          description: The type of the trigger
+          type: string
+          enum: ["explicit", "webhook"]
```

Comment:
These don't match the internal data, should be a 1-1 mapping between them though, and I feel consistency with the config values is important here.

Reply:
Should we rename `explicit` to `api`? I think it's a little more self-explanatory, but I'm open to other arguments.

Reply:
We have some exploratory config documentation suggesting `explicit` over `api`. I'm not wedded to either name but do feel we need to be consistent with the value used in config.